src/site/navigation/facility_sidebar_nav.drupal.liquid

```diff
@@ -1,38 +1,42 @@
-<nav id="va-detailpage-sidebar" data-drupal-sidebar class="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav va-c-facility-sidebar">
```

Comment:
I think there are some styles that use `va-c-facility-sidebar`. Do those need to be removed or updated?

Reply:
the class is still there, it was just listed twice. This commit removes the second occurrence.

Reply:
Ha, so it is.